<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wikingerschach</string>
  </property>
  <widget class="QPushButton" name="btn_plus1_t1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_plus1_t2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_plus3_t2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_plus3_t1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_plus5_t2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>380</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_plus5_t1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set15_t2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>180</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt; 15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set15_t1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt; 15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set0_t2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>280</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set0_t1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_undo_t2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>380</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_undo_t1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_team1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Team 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_team2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Team 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Points: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Points: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_score_team1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 170, 0)</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_score_team2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 170, 0)</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
